cmd/opentelemetry/app/exporter/elasticsearchexporter/esmodeltranslator/modeltranslator.go

```diff
@@ -248,7 +248,7 @@ func traceIDToString(high, low uint64) string {
 }
 
 func (c *Translator) process(resource pdata.Resource) *dbmodel.Process {
-	if resource.IsNil() || resource.Attributes().Len() == 0 {
+	if resource.Attributes().Len() == 0 {
```

Comment:
It seems that `IsNil()` was removed and the resource is guaranteed to never be nil?

I find these generate proto wrappers very confusing.  Do you have any insight into why this was removed?

Reply:
Well, that went from my issue about panic in marshalling https://github.com/open-telemetry/opentelemetry-collector/issues/1985. Then gradually all proto things became non-nullable. I believe, that using pointers was a mistake from the very beginning

Reply:
Yes, you're right, resource now cannot be nil